src/s1reader/s1_reader.py

```diff
@@ -279,6 +285,30 @@ def get_ipf_version(tree: ET):
 
     return ipf_version
 
+def get_path_aux_cal(directory_aux_cal: str, str_platform: str, instrument_cfg_id: int):
+    '''
+    Decide which aux_cal to load
+
+    Parameters:
+    -----------
+    diectory_aux_cal: str
+        Directory for the AUX_CAL .zip files
+    str_platform: str
+        'S1A' or 'S1B'
+    instrument_cfg_id: int
+        Instrument configuration ID
+
+    Returns:
+    --------
+    path_aux_cal: AUX_CAL file that corresponds to the criteria provided
+
+    '''
+    list_aux_cal = glob.glob(f'{directory_aux_cal}/{str_platform}_AUX_CAL_V*.SAFE.zip')
+    list_aux_cal.sort()
+    path_aux_cal = list_aux_cal[instrument_cfg_id-1]
+
+    return path_aux_cal
+
 def is_eap_correction_necessary(ipf_version: version.Version) -> SimpleNamespace :
     '''Examines if what level of EAP correction is necessary, based on the IPF version.
```

Comment:
What is EAP? Please define the acronym here.

Reply:
It's defined in the docstring in `s1_annotation.py`, but also added the definition in `s1_reader.py` as well.